packages/main/src/Carousel.js

```diff
@@ -197,7 +185,7 @@ const metadata = {
 	events: /** @lends sap.ui.webcomponents.main.Carousel.prototype */ {
 
 		/**
-		 * Fired whenever the <code>selectedIndex</code> changes due to user interaction,
+		 * Fired whenever the <code>selectedIndex</code> property changes due to user interaction,
```

Comment:
`selectedIndex` is now private, so this explanation here is not clear

Reply:
Reworded it